CODIGO/frmBancoCuenta.frm

```diff
@@ -468,7 +468,6 @@ Private Sub Form_Unload(Cancel As Integer)
     
     On Error GoTo Form_Unload_Err
     
```

Comment:
why remove this?

Reply:
We should not play sounds on `Form_Unload`

Reply:
I understand it might not be the best place, maybe we should move this function to the close button or any other interaction that its calling the form unload but we should keep the functionality

Reply:
No, I don't think it's a good idea because there is no direct control when the form is Unloaded, this can happen after the audio system was shutdown and will create problems.

We must not put this kind of code in `Form_Unload`